<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintWidget</class>
 <widget class="QWidget" name="PrintWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>541</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:orange</string>
   </property>
  </widget>
  <widget class="QLabel" name="printDialog_logo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_print">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>83</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Tables</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
